components/api/gen/schema.bal

```diff
@@ -0,0 +1,96 @@
+// ------------------------------------------------------------------------
```

Comment:
Let's move this to `core` module (components/api/core/gen directory)

Reply:
This may cause the cyclic import issue. Hence kept in the same location.